### DISCUSSION TITLE
Colors for indent-rainbow?

### DISCUSSION THREAD

**Discussion by lampewebdev** [ykrx/Lightbulb#2] [General]
Hey,

has someone maybe a good setup for [indent-rainbow](https://marketplace.visualstudio.com/items?itemName=oderwat.indent-rainbow) ?

**Comment by ykrx** [comment]
Just added this as a section under "Setup" in the README. These are the colors I use:

```json
"indentRainbow.colors": [
    "rgba(141, 151, 170, 0.0)",
    "rgba(141, 151, 170, 0.035)"
],
"indentRainbow.errorColor": "#f98d7c29",
"indentRainbow.indicatorStyle": "classic",
```

Thanks for bringing this up @lampewebdev!

**Reply by lampewebdev** [reply]
Thank you!
I will add this :) 

Great Theme! Its now my daily driver!

**Reply by lampewebdev** [reply]
<img width="837" alt="image" src="https://user-images.githubusercontent.com/44268468/200073561-0a80da97-11d2-42ed-b9b0-09e1d32bab38.png">
Looks really good now 🥇
